src/pages/functional-testing-framework/metadata.md

```diff
@@ -264,6 +268,154 @@ _Catalog/Data/CategoryData.xml_:
+    <data key="sales_representative_id">1</data>
+    <var key="super_user_id" entityType="customer" entityKey="id"/>
+    <var key="company_email" entityType="customer" entityKey="email"/>
+</entity>
+<entity name="NewCompany" type="company2">
+    <data key="status">1</data>
+    <data key="company_name" unique="suffix">New Company</data>
+    <array key="street">
+        <item>90 Schevche street</item>
+        <item>Bld New</item>
+    </array>
+    <data key="city">Mukachevo</data>
+    <data key="country_id">VA</data>
+    <data key="region">New region</data>
+    <data key="postcode">89600</data>
+    <data key="telephone">512-353-3333</data>
+    <data key="customer_group_id">1</data>
+    <data key="sales_representative_id">1</data>
+    <var key="super_user_id" entityType="customer" entityKey="id"/>
+    <var key="company_email" entityType="customer" entityKey="email"/>
+</entity>
+```
+
+Notice that the 2 company entities have 2 different types `type="company"` and `type="company2"`.  These types will be used later in _Data_ fields and _Metadata_ urls for company relations.
+
+_CompanyRelation/Data/CompanyRelationsData.xml_:
+
+```xml
+<entity name="EstablishOneCompanyRelationData" type="company_relations">
+    <requiredEntity type="company_relation">CompanyRelationData</requiredEntity>
+</entity>
+```
+
+- `type="company_relations"` points to the operation with `dataType=company_relations"` and `type="create"` in the _Metadata_ directory.
+- `<requiredEntity type="company_relation">CompanyRelationData</requiredEntity>` points to the data entity with `name="CompanyRelationData"` and `type="company_relation"`.
+
+_CompanyRelations/Data/CompanyIdsData.xml_:
+
+```xml
+<entity name="CompanyRelationData" type="company_relation">
+    <var key="company_id" entityType="company2" entityKey="id"/>
+</entity>
+```
+
+- `type="company_relation"` points to the operation with `dataType=company_relation"` and `type="create"` in the _Metadata_ directory.
+- The `company_id` will only get the `id` from the company with `type="company2"`
+
+_CompanyRelation/Metadata/CompanyIdsMeta.xml_:
+
+```xml
+<operation name="CreateCompanyRelation" dataType="company_relation" type="create" >
+    <field key="company_id">integer</field>
+</operation>
+```
+
+`<field key="company_id">integer</field>` will create an object with a key `company_id` with a value of type integer.
+```json
+{
+  "company_id": 1
+}
+```
+
+_CompanyRelation/Metadata/CompanyRelationsMeta.xml_:
+
+```xml
+<operation name="CreateCompanyRelations" dataType="company_relations" type="create" auth="adminOauth" url="/V1/company/{company.id}/relations" method="POST">
+    <contentType>application/json</contentType>
+    <array key="relations">
+        <value>company_relation</value>
+    </array>
+</operation>
+```
+
+As a result, the Functional Testing Framework sends a POST request with an array of company_ids in the body to the `https://example.com/rest/V1/company/:parentId/relations`.
```

Comment:
> https://example.com/rest/V1/company/:parentId/relations

change to "https://example.com/rest/V1/company/{company.id}/relations"

Reply:
fixed